### PR TITLE
WebApp: Added eslinter and made changes according to current linter configurations

### PR DESCRIPTION
**Pull request by aksh-konda**
Added eslinter and according to the current configurations, there were a lot of modifications done. 
Can you please suggest any required configurations for the linter?.
thank you.

### CODE REVIEW

File: app-collab-note/src/App.css [hunk 1]
```diff
@@ -5,7 +5,7 @@
 }
 
 .App {
-	font-family: 'Open Sans', sans-serif;
+	  font-family: 'Open Sans', sans-serif;
```
Comment (roman-r-m) [inline app-collab-note/src/App.css]: is this change made by the linter? looks strange to me

**Review by roman-r-m — APPROVED** [review]

File: app-collab-note/src/App.css (hunk 1 above)
Comment (Aksh-Konda) [reply app-collab-note/src/App.css:8]: I might have formatted the document in vscode.😅

Comment (roman-r-m) [reply app-collab-note/src/App.css]: looks like it was just one line, not the whole doc

**Review by laurent22 — APPROVED** [review]
Looks good. For the eslint config, the best is to extend the recommended settings as you've done, especially since they'll include rules about code correctness. Later on if you find that some rules are too strict or get in the way, you can selectively disable them.

File: app-collab-note/src/App.css (hunk 1 above)
Comment (aksh-konda) [reply app-collab-note/src/App.css]: Sorry, I didn't intend for it to be in the commit. Changed it now.

**Pull request merged**